src/Mapping/Driver/AbstractAnnotationDriver.php

```diff
@@ -24,6 +25,9 @@ abstract class AbstractAnnotationDriver implements AnnotationDriverInterface
      * Annotation reader instance
      *
      * @var object
+     * @phpstan-var Reader|AttributeReader
```

Comment:
There is no special case here for `@phpstan-var`. Conventional tags can be used:
```suggestion
     * @var Reader|AttributeReader
```